recipes/xpore/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "xpore" %}
+{% set version = "0.5.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2478c434f40aa24e901fc177a726f2f90fbf09b87ac530cdbc23908729c907be
+
+build:
+  number: 0
+  noarch: generic
+  entry_points:
+    - xpore-dataprep=xpore.scripts.dataprep:main
+    - xpore-diffmod=xpore.scripts.diffmod:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - h5py >=2.10.0
+    - numpy >=1.18.0
+    - pandas >=0.25.3
+    - pip
+    - pyensembl >=1.8.5
+    - python
+    - pyyaml
+    - scipy >=1.4.1
+  run:
+    - h5py >=2.10.0
+    - numpy >=1.18.0
+    - pandas >=0.25.3
+    - pyensembl >=1.8.5
+    - python
+    - pyyaml
+    - scipy >=1.4.1
+
+test:
+  imports:
+    - xpore
+    - xpore.diffmod
+    - xpore.scripts
+    - xpore.utils
+  commands:
+    - xpore-dataprep --help
+    - xpore-diffmod --help
+
+about:
+  home: "https://github.com/GoekeLab/xpore"
+  license: MIT
+  license_family: MIT
+  license_file:
```

Comment:
can you please add a license file?

Reply:
stated "LICENCE" as in the dev github repo. Am I required to add here the *file* ?